src/main/scala/scorex/core/transaction/box/Box.scala

```diff
@@ -12,6 +13,7 @@ trait Box[P <: Proposition] extends BytesSerializable {
   val proposition: P
 
   val id: ADKey
+  val json: Json
```

Comment:
Do we need to add this method here?

Probably we should be consistent. Either we have a `json` method in all our abstract data traits (e.g. also in `Transaction`,`Block`, ...) or in none of them. (cc: @kushti)